pypika/queries.py

```diff
@@ -333,6 +333,15 @@ def __init__(self, quote_char='"', dialect=None, wrap_union_queries=True):
         self.dialect = dialect
         self.wrap_union_queries = wrap_union_queries
 
+    def __copy__(self):
+        newone = type(self)()
+        newone.__dict__.update(self.__dict__)
+        for attr in self.__dict__.keys():
+            value = self.__dict__[attr]
+            if isinstance(value, list):
```

Comment:
Is there also a case when the value can be another collection e.g. a tuple, or is it always a list?

Reply:
well in the QueryBuilder class there is one attribute that is a set, so I'll add that here. Tuples are immutable, so it wouldn't be a problem to share them across copies of a query

Reply:
Ok, as long as that collection always stores one level of immutable objects that is fine. I didn't know if it may store something mutable e.g. a nested list which you'd need to deep copy.